### PR TITLE
Fix Makefile

### PR DESCRIPTION
**Pull request by Serabe**
Fixes CI repo and dependencies in macOS

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -81,19 +78,19 @@ hugo-dependencies:
...
 		$(MKDIR) $(HUGO_PATH); \
 		cd $(HUGO_PATH); \
-		ext="zip"; \
-		if [ "$(ARCH)" == "linux" ]; then ext="tar.gz"; fi; \
+		ext="tar.gz"; \
+		if [ "$(ARCH)" == "windows" ]; then ext="zip"; fi; \
 		file="hugo.$${ext}"; \
 		$(CURL) https://$(HUGO_URL)/releases/download/v$(HUGO_VERSION)/$(HUGO_URL_NAME).$${ext} -o $${file}; \
-		if [ "$(ARCH)" == "linux" ]; then tar -xvzf $${file}; else unzip $${file}; fi; \
+		if [ "$(ARCH)" == "windows" ]; then unzip $${file}; else tar -xvzf $${file}; fi; \
```
Comment (dpordomingo) [inline Makefile]: why do we support Windows?

Comment (erizocosmico) [inline Makefile]: This was initially part of the autohugo project, which was a generic thing. Now that it's for us idk how much value adds to support a platform we are never ever using.

Comment (Serabe) [inline Makefile]: But autohugo is dead, isn't it?

**Comment by erizocosmico** [comment]
Whether it is or not, it is irrelevant for this project, so I'd say kill windows is an option.

El vie., 9 jun. 2017 a las 15:06, Sergio Arbeo (<notifications@github.com<mailto:notifications@github.com>>) escribió:

@Serabe commented on this pull request.

________________________________

In Makefile<https://github.com/src-d/new_landing/pull/10#discussion_r121120950>:

>               file="hugo.$${ext}"; \
                $(CURL) https://$(HUGO_URL)/releases/download/v$(HUGO_VERSION)/$(HUGO_URL_NAME).$${ext} -o $${file}; \
-               if [ "$(ARCH)" == "linux" ]; then tar -xvzf $${file}; else unzip $${file}; fi; \
+               if [ "$(ARCH)" == "windows" ]; then unzip $${file}; else tar -xvzf $${file}; fi; \


But autohugo is dead, isn't it?

File: Makefile (hunk 1 above)
Comment (dpordomingo) [inline Makefile]: I'd drop the window support from the Landing

**Comment by dpordomingo** [comment]
@Serabe Can you drop windows support in a new PR?

**Pull request merged**